Samples/PowerShell/functions/func_ModernDashboards.ps1

```diff
@@ -4,6 +4,134 @@
 #Requires -Module @{ ModuleName = 'SwisPowerShell'; ModuleVersion = '3.0.0' }
 #Requires -Version 7
 
+<#
+.Synopsis
+    List Modern Dashboards from SolarWinds Orion system
+.DESCRIPTION
+    Connects to SolarWinds Information Service and enumerates the user-defined Modern Dashboards
+.EXAMPLE
+    $SwisConnection = Connect-Swis -Hostname "192.168.11.165" -Username "admin" -Password "MyComplexPassword"
+    PS C:\Exports> Get-SwisModernDashboard -SwisConnection $SwisConnection
+
+    This command list all of the Modern Dashboards
+.EXAMPLE
+    $SwisConnection = Connect-Swis -Hostname "192.168.11.165" -Username "admin" -Password "MyComplexPassword"
+    PS C:\> Get-ModernDashboard -SwisConnection $SwisConnection -DashboardId 9 
+
+    Lists Modern Dashboard with ID 9
+.EXAMPLE
+    $SwisConnection = Connect-Swis -Hostname "192.168.11.165" -Username "admin" -Password "MyComplexPassword"
+    PS C:\> Get-ModernDashboard -SwisConnection $SwisConnection -DashboardId 1 
+
+    PS C:\> Get-ModernDashboard -SwisConnection $SwisConnection -DashboardId 1 -IncludeSystem
+
+    The first command will (most likely) return nothing as DashboardId 1 is a "system" dashboard
+    The second command will return the details of Dashboard ID 1
+
+.NOTES
+    Author:  Kevin M. Sparenberg (https://thwack.solarwinds.com/members/kmsigma)
+    Version: 0.9
+    Last Updated: 2021-09-30
+    Validated: Orion Platform 2020.2.5
+
+    TBD List:
+        * Validate that the script works on non-Windows
```

Comment:
Maybe just macOS, since you have validated on Linux?

Reply:
Yeah - good call.  I'll make the update.

Reply:
Also making the update on the other functions as well.